gapii/cc/vulkan_extras.inc

```diff
@@ -161,3 +161,47 @@ void walkImageSubRng(
 void parseShaderModule(
     StageData* stage,
     gapil::Ref<DescriptorInfo>& descriptors);
+
+// TODO: does this need a mutex?
+std::map<VkCommandBuffer, std::vector<VkBufferMemoryBarrier>> mExternalBufferBarriers;
```

Comment:
Please std::unordered_map, unless we REALLY need command-buffer handles to be ordered.

Reply:
Done.